plugins/modules/win_acl.py

```diff
@@ -123,4 +124,12 @@
     rights: Read,Write,Modify,FullControl,Delete
     type: deny
     state: present
+
+- name: Set certificate private key FullControl to Fed-Phil
+  ansible.windows.win_acl:
+    path: Cert:\LocalMachine\My\168ba8c488463f88c6648466a22484b6189e165f
+    user: Fed-Phil
+    type: allow
+    state: present
+    rights: FullControl
```

Comment:
Am I right in seeing that only `Read` and `FullControl` is implemented for certificate rights? They are the only ones implemented in the UI but it would be good for our docs to state that they are the only ones valid for certificate private keys.

Reply:
So more than what is documented on line 48?